devices/danfoss.js

```diff
@@ -26,7 +26,8 @@ module.exports = [
             tz.thermostat_keypad_lockout, tz.thermostat_system_mode, tz.danfoss_load_balancing_enable, tz.danfoss_load_room_mean,
             tz.thermostat_weekly_schedule, tz.thermostat_clear_weekly_schedule, tz.thermostat_programming_operation_mode,
             tz.danfoss_window_open_feature, tz.danfoss_preheat_status, tz.danfoss_adaptation_status, tz.danfoss_adaptation_settings,
-            tz.danfoss_adaptation_control, tz.danfoss_regulation_setpoint_offset],
+            tz.danfoss_adaptation_control, tz.danfoss_regulation_setpoint_offset,
+            tz.danfoss_thermostat_occupied_heating_scheduled_setpoint],
```

Comment:
Please also add an `exposes.numeric` for this in the list below (such that you can easily set this value from the frontend under the exposes tab)

Reply:
@Koenkk I tried this in this branch [danfoss_expose_scheduled_setpoint](https://github.com/Koenkk/zigbee-herdsman-converters/compare/master...glad-wolf:zigbee-herdsman-converters:danfoss_expose_scheduled_setpoint) but it does not work well because it modifies the same value as the already exposed `occupied_heating_setpoint` does.

The new exposed value can be used to change the setpoint but it does not always reflects the current setpoint.

Reply:
@Koenkk I renamed the option as suggested. And I extended fromZigbee to also update the new option. Now it works well (also with ea.All).